plugin.video.embycon/resources/lib/play_utils.py

```diff
@@ -895,7 +942,11 @@ def onNotification(self, sender, method, data):
     def screensaver_activated(self):
         log.debug("Screen Saver Activated")
 
-        xbmc.executebuiltin("Dialog.Close(selectdialog, true)")
+        # stop playback when switching users
+        if xbmc.Player().isPlaying():
```

Comment:
```suggestion
        if xbmc.getCondVisibility("Player.Playing"):
            xbmc.executebuiltin("PlayerControl(stop)")
--



```

Reply:
Since you do not need the player instance you can avoid creating 2 player instances by using boolean conditions and the respective builtin function

Reply:
Is there a problem using Player()?
How about this instead

`        player = xbmc.Player()
        if player.isPlaying():
            player.stop()`

Reply:
All great info. I will look into this for the next release.
If we can release/merge this version now as it has passed testing and QA that would be good.